_docs/tasks/security/authn-policy.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Basic Istio Authentication Policy
+overview: Shows you how to use Istio authentication policy to setup mutual TLS and simple end-user authentication.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+Through this task, you will learn how to:
+
+* Use authentication policy to setup mutual TLS.
+
+* Use authentication policy to do end-user authentication.
+
+
+## Before you begin
+
+* Understand Isio [authentication policy]({{home}}/docs/concepts/security/authn-policy.html) and related [mutual TLS authentication]({{home}}/docs/concepts/security/mutual-tls.html) concepts.
+
+* Know how to verify mTLS setup (recommend to walk through [testing Istio mutual TLS authentication]({{home}}/docs/tasks/security/mutual-tls.html))
+
+* Have a Kubernetes cluster with Istio installed, without mTLS. See [the Istio installation task]({{home}}/docs/setup/kubernetes/quick-start.html) and follow step 5a.
+
+* For demo, create two namespaces `foo` and `bar`, and deploy [httpbin](https://github.com/istio/istio/tree/master/samples/httpbin) and [sleep](https://github.com/istio/istio/tree/master/samples/sleep) with sidecar on both of them. Also, run another sleep app without sidecar (to keep it separate, run it in `legacy` namespace)
+
+```bash
+kubectl create ns foo
+kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml) -n foo
+kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml) -n foo
+
+kubectl create ns bar
+kubectl apply -f <(istioctl kube-inject -f samples/httpbin/httpbin.yaml) -n bar
+kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml) -n bar
+
+kubectl create ns legacy
+kubectl apply -f samples/sleep/sleep.yaml -n legacy
+
+```
+
+* Verifying setup by sending an http request (using curl command) from any sleep pod (among those in namespace `foo`, `bar` or `legacy`) to either `httpbin.foo` or `httpbin.bar`. All requests should success with http code 200.
+
+For example, here is a command to check `sleep.bar` to `httpbin.foo` reachability:
```

Comment:
If the tex below is supposed to be under the above bullet item, it all needs to be indented by 3.

Reply:
Done